README.md

```diff
@@ -133,12 +143,207 @@ Want to help out? Pull requests (PRs) are always welcome! If you want to help ou
+
+```YAML
+---
+title: "Get started"
+description: "The Filecoin Network is made with storage providers and clients. They make deals and contribute to maintaining the Filecoin blockchain, obtaining storage services, and receiving rewards in the process. This section walks you through how to get started, build a node, and create a simple application."
+lead: "The Filecoin Network is made with storage providers and clients. They make deals and contribute to maintaining the Filecoin blockchain, obtaining storage services, and receiving rewards in the process. This section walks your through how to get started, build a node, and create a simple application."
+menu:
+    getstarted:
+        parent: "getstarted-overview"
+aliases:
+    - /get-started
+    - /how-to/install-filecoin
+---
+```
+
+It's also good to note that we use the YAML as our front-matter format. We could use [JSON or TOML](https://gohugo.io/content-management/front-matter#front-matter-formats) if we really wanted, but we found YAML the easiest to read. Plus, _yammal_ is fun to say.
+
+This list has been created in order of commonality; variables you will come across most often are closer to the top of this list.
+
+#### Title
+
+The `title` variable defines what the `<h1>` tag on this page will say, along with the contents of `<title>` in this page's `<head>`. This variable also defines what is shown as the sidebar item; however, this can be overwritten in the `menus` config file.
+
+```YAML
+---
+title: "Get started"
+---
+```
+
+![](./static/images/front-matter-variables-title.png)
+
+#### Description
+
+The `description` variable defines what is in the [meta `<description>`](https://moz.com/learn/seo/meta-description) tag within the `<head>` tag of this page's HTML.  This description often shows up in search engine results, and social network embeds. This description is meant to give the reader an idea of the content on this page and how it relates to their search query.
+
+```YAML
+---
+description: "The Filecoin Network is made with storage providers and clients. They make deals and contribute to [...]"
+---
+```
+
+![](./static/images/front-matter-variables-description.png)
+
+#### Lead
+
+The `lead` variable defines the content of the first paragraph on a page. This is usually an introduction, informing the reader what this page is referring to, what they're about to learn, and any prerequisites for understanding the content on this page. Often, the content of this variable is the same as the `description` variable.
+
+```YAML
+---
+lead: "The Filecoin Network is made with storage providers and clients. They make deals and contribute to [...]"
+---
+```
+
+![](./static/images/front-matter-variables-lead.png)
+
+#### Weight

```

Comment:
Do we have certain rules to choose the weight for each page? For example, we should only increase the number by 10, then use 10, 20, 30, 40. 
Because I was not sure if I should use 15 or 11 after 10.

Reply:
There's not strict _rules_. But it's just easier to use multiples of 10. It makes it easier to slot in new pages if we need them in the future. For example: if we want a new section between **What is Filecoin** and **How Filecoin works** then we can give it a weight of 15, instead of having to shuffle everything around.

Does that make sense?